app/controllers/mixins/manager_controller_mixin.rb

```diff
@@ -265,7 +265,7 @@ def delete
                      "Delete initiated for %{count} Providers",
                      providers.length) % {:count => providers.length})
       end
-      replace_right_cell
+      replace_right_cell(:replace_trees => [x_active_accord])
```

Comment:
@lgalis do you think this line should be removed?

Reply:
No, I think this change should have stayed, but removed it nonetheless as it is not related directly to the problem reported in the BZ.